.github/workflows/SyntaxKit.yml

```
@@ -41,47 +46,58 @@
+            runs-on: macos-15
+            xcode: "/Applications/Xcode_26.0.app"
                       
           # iOS Build Matrix
           - type: ios
             runs-on: macos-15
-            xcode: "/Applications/Xcode_16.4.app"
+            xcode: "/Applications/Xcode_26.0.app"
             deviceName: "iPhone 16 Pro"
-            osVersion: "18.5"
+            osVersion: "26.0"
   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)